resources/views/components/content-area.blade.php

```diff
@@ -16,7 +16,7 @@
                     <div class="slideout-main-menu mt:hidden">
                         <ul class="main-menu mb-2">
                             <li>
-                                <a role="button" class="main-menu-toggle pt-2 pb-2 pl-3 pr-3 block" tabindex="0" aria-expanded="false">Main Menu</a>
+                                <a role="button" class="main-menu-toggle pt-2 pb-2 pl-3 pr-3 block" tabindex="0" aria-expanded="false">{{ config('base.top_menu_label') }}</a>
```

Comment:
You could use short site tile or base config here if interested!

Reply:
Good point. I've found a few in the CMS that is not actually short and a replica of the full site title. Ideally, this would be less than 16 chars. If that limit existed in the CMS I'd be totally comfortable.

Also, the dealbreaker to using short title field is that this label is meant to be displayed on the subsites (which wouldn't have an accurate short title for this context).

Let's leave it in code for the moment and have the discussion if it becomes an issue.